packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -92,6 +92,25 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
         this.setState({ activePaymentMethod: null });
     }
 
+    private onOrderCancel = data => {
+        if (this.props.onOrderCancel) {
+            return new Promise(() => {
+                return this.props.onOrderCancel(data);
+            });
```

Comment:
Why do we need to do `new Promise` here? Can't just call the `this.props.onOrderCancel` ?